### DISCUSSION TITLE
Maybe, the site needs a separate forum? / Возможно, сайту требуется отдельный форум?

### DISCUSSION THREAD

**Discussion by yuriy-zhilovets2** [PastVu/pastvu#553] [Ideas / Идеи и предложения]
Сейчас обсуждение вопросов, не связанных прямо с фотографиями, идет на выделенной странице новостей, которая уже разрослась до приличных размеров с весьма сомнительной при этом эргономикой. Возможно, удобнее будет поставить на отдельном домене готовый форум типа phpBB2, использующий логины с сайта?

**Comment by kabalin** [comment]
Перенес в дискуссии.
